### ISSUE TITLE
Bottom next/previous navigation bar is enclosed in h3 tag

### ISSUE THREAD

**Issue by ShubhamOulkar** [expressjs/expressjs.com#2056]
<img width="563" height="142" alt="Image" src="https://github.com/user-attachments/assets/8b845c3c-a702-4261-af9e-1cdc2b6ddb13" />

Now next and previous links are enclosed in `h3` tag as shown above, expected to be enclosed in a `nav` container and proper styling is required.

New Edit:- 
I want to extend this task as follows.
We should write `a reusable bottom navigation bar` so that we can use it on guide, advance topics and resources pages.

**Comment by sanajitjana** [comment]
Hey @ShubhamOulkar, I am interested to work on this, can you assign me please?

**Comment by ShubhamOulkar** [comment]
@sanajitjana we don't assign issues, but you can work on it.

**Comment by CHINMAYJAI** [comment]
hii, can you explain more what you want with the expected result screenshots so that i will be more clear

**Comment by ShubhamOulkar** [comment]
I can not provide you screenshot, but this should be enclosed in `<nav>`

**Comment by CHINMAYJAI** [comment]
> I can not provide you screenshot, but this should be enclosed in `<nav>`

you mean dom reconstruction?

**Comment by ShubhamOulkar** [comment]
> you mean dom reconstruction?

Not complete DOM reconstruction but redesign next and previous link components.

**Comment by CHINMAYJAI** [comment]
> > you mean dom reconstruction?
> 
> Not complete DOM reconstruction but redesign next and previous link components.

<img width="948" height="317" alt="Image" src="https://github.com/user-attachments/assets/aced88b0-0e16-4a44-ad4c-a9a17d3a6e9f" />

you need like this?
